packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md

```diff
@@ -0,0 +1,200 @@
+
+</div>
+
+<!-- Source for image. Generated using http://www.plantuml.com/plantuml/uml/
+skinparam monochrome true
+participant "Service app (customer instance)" as client
+participant "Authorization Server (customer Okta org)" as okta
+participant "Resource Server (customer Okta APIs)" as app
+client -> okta: Access token request to /token
+okta -> client: Access token response
+client -> app: Request with access token
+app -> client: Response
+-->
+
+## Build your API service integration
+
+### Collect and save customer credentials
+
+Okta generates a unique set of credentials (client ID and client secret) for your customer when they register your integration in their Okta org. See [HOC&mdash;Set up an API service integration](https://www.figma.com/proto/1hqMxP4MrqXR07atucaGvM/OAuth-API-Integrations---Partner-Mocks?node-id=1550%3A9898&starting-point-node-id=1550%3A9668). You must collect and store these credentials for each customer to allow your integration to work with the customer's Okta org.
+
+To implement the Client Credentials grant flow in your integration, provide an interface to collect and store these API service integration credentials:
+
+* Okta tenant (organization) domain (for example, `acme.okta.com`)
+* client ID
+* client secret
+
+> **Note:** This guide refers to these domain and credentials as `${customerOktaDomain}`, `${client_id}`, and `${client_secret}` variables.
+
+### Selecting scopes
+
+You must specify the resources and actions that your API service app requires to do its job. Use the least-privilege principle and only specify the minimal scopes required. See the list of available Okta [Scopes and supported endpoints](/docs/guides/implement-oauth-for-okta/main/#scopes-and-supported-endpoints).
+
+There are two types of scope: read and manage. Read scopes can only view resources, while manage scopes can read, create, update, and delete resources. You don't need to request a read scope when you request a manage scope because manage scopes already includes read access.
+
+| Action    | Read scopes           | Manage scopes   |
+| ----------- | -------------- | ------------- |
+| Read | Yes | Yes |
+| Create | No | Yes |
+| Update | No | Yes |
+| Delete | No | Yes |
+
+##### Silent downscoping
+
+The Okta Org Authorization Server returns all scopes that you request if you registered those scopes along with your integration. Currently, API service integrations don't support optional scopes. You can request a subset of your-integration-supported scopes when requesting an access token from the `/token` endpoint. For example, if you registered the `okta.users.manage`, `okta.groups.manage`, and `okta.apps.manage` scopes for your integration, but your service only needs to retrieve and update Okta groups for a specific task, then your access token request only requires the `okta.groups.manage` scope.
```

Comment:
Like the example! 

Maybe we could say something like 

> then you can request only the `okta.groups.manage` scope when you access the token endpoint

?

Reply:
Updated section. Please review